overlord/snapstate/snapstate_test.go

```diff
@@ -2754,6 +2754,126 @@ func (s *snapmgrTestSuite) TestInstalling(c *C) {
 	c.Check(snapstate.Installing(s.state), Equals, true)
 }
 
+func (s *snapmgrTestSuite) TestUpdateAmendRunThrough(c *C) {
+	si := snap.SideInfo{
+		RealName: "some-snap",
+		Revision: snap.R(7),
```

Comment:
realistically this would be a local revision instead

Reply:
ah, drat. I'll fix it (maybe in a followup if no other things need changing)